kroxylicious-filters/kroxylicious-authorization/src/main/java/io/kroxylicious/filter/authorization/MetadataEnforcement.java

```diff
@@ -111,6 +117,11 @@ private CompletionStage<RequestFilterResult> onNonIdempotentMetadataRequest(Requ
         return context.sendRequest(initialRequestHeader, initialRequest)
                 .thenCompose(notCreateResponse -> {
                     var notCreateMetadataResponse = (MetadataResponseData) notCreateResponse;
+                    Errors error = Errors.forCode(notCreateMetadataResponse.errorCode());
+                    if (error != Errors.NONE) {
+                        LOG.info("Internal metadata response from broker has error code {}", error);
```

Comment:
Might be helpful to include `context.sessionId` here as well, Yes I know it should be there automagically but we don't live in that world yet...